spotify_project/albums/migrations/0001_initial.py

```diff
@@ -15,9 +15,15 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Album',
             fields=[
-                ('spotify_id', models.CharField(max_length=22, primary_key=True, serialize=False)),
```

Comment:
I highly suggest you tell black and flake8 to skip migrations. They are auto generated files. Which it's poor form to post process like this.

Reply:
oooh, you're up and providing good advice!

Reply:
Lol, yea. I am getting ready to go to bed though, so your about to be on your own again.